cilium/cmd/kvstore.go

```diff
@@ -55,6 +55,6 @@ func setupKvstore(ctx context.Context) {
 func init() {
 	rootCmd.AddCommand(kvstoreCmd)
 	flags := kvstoreCmd.PersistentFlags()
-	flags.StringVar(&kvStore, "kvstore", "", "kvstore type")
-	flags.Var(option.NewNamedMapOptions("kvstore-opts", &kvStoreOpts, nil), "kvstore-opt", "kvstore options")
+	flags.StringVar(&kvStore, "kvstore", "", "Kvstore type")
```

Comment:
Same here, `Kvstore` doesn't really make sense, this stands for "Key-Value Store". Probably this should be expanded out like that.